contrib/scripts/kind.sh

```diff
@@ -9,7 +9,15 @@ default_workers=1
 default_cluster_name=""
 default_image=""
 default_kubeproxy_mode="iptables"
-default_ipfamily="dual"
```

Comment:
@bimmlerd 

I do not think we can keep this default value, since user is supposed to apply this config via env value `IPFAMILY`?

Reply:
default falls back to `dual` as implemented, what makes sense if we have `default_ipfamily="dual"` before the if statement? Or are you saying we should have "none" as default and if not specified we should go to the if statement? so that we can see if `default_ipfamily` is explicitly specified by user? i thought in that case, user is expected to use `IPFAMILY` instead.

Reply:
Sorry, I wasn't super clear.
I as thinking it's more elegant to keep

    default_ipfamily="dual"

and then somewhere further down

    if [ "$(uname 2>/dev/null)" == "Linux" ] && [ $(sysctl -n net.ipv6.conf.all.disable_ipv6) == 1 ] ; then
      default_ipfamily="ipv4"
    fi

But it doesn't really matter :)